docs/resources/cluster.md

```diff
@@ -47,7 +47,7 @@ resource "databricks_cluster" "shared_autoscaling" {
 
 The following example demonstrates how to create an autoscaling cluster with [Delta Cache](https://docs.databricks.com/delta/optimizations/delta-cache.html) enabled:
 
-```hcl
```

Comment:
non-hcl markdown blocks are displayed differently on tf registry. try this file in https://registry.terraform.io/tools/doc-preview doc preview tool

Reply:
yeah I fixed that, in another PR because I didn't like the poorly named auto-branch that GitHub gave it.. pedantic I know 🙂